### ISSUE TITLE
[CAIA Intake] Benefits Disability Experience 526 EZ: Toxic Exposure Screens

### ISSUE THREAD

**Issue by Kate-allbee** [department-of-veterans-affairs/va.gov-team#71168]
## About your team

- **Team name:** Benefits Disability Experience 526ez Team 1
- **OCTO-DE product owner:** `NO INFO. PROVIDED`
- **Product manager**: Rakshinda Aslam
- **Designer**:  Kate Allbee
- **Accessibility specialist:**  `N/A`
- **Content writer:** `N/A`
- **Slack channel**:[#benefits-disability-design](https://dsva.slack.com/archives/C053UDWMH7U/p1698182576413519)
 
## CAIA Support Request

We'd like to get CAIA's support in reviewing our most recent content for the Toxic Exposure screens. We have made changes post-research and would love feedback if we're on the right track and it follows the content guide.

### Examples
- We’re updating a form and want to confirm we’re doing it correctly.
- We’re launching a new online tool to VA.gov and need support with our user flows.
- We need to make sure the content we’re using in our tool is written in plain language and follows the style guide.

### Additional Information
- **Timeframe**: We expect to launch our product to 100% of users on: Q1 of 2024
- **Collaboration Cycle** [Ticket 60140 opened June 12, 2023](https://app.zenhub.com/workspaces/disability-experience-63dbdb0a401c4400119d3a44/issues/gh/department-of-veterans-affairs/va.gov-team/60140)
- **Next step in the Collab Cycle:** Staging

### Available Files as of Kickoff 12/14/2023

- [Product outline](https://github.com/department-of-veterans-affairs/va.govteam/blob/master/products/disability/526ez/README.md)
- [Design prototype](https://www.figma.com/proto/dBKlB23Hs2oa53euXcXmmK/526-toxic-exposure?page-id=0%3A411&type=design&node-id=0-1296&viewport=-915%2C-532%2C0.18&t=t5oRHxPHWhARJWdb-1&scaling=min-zoom&starting-point-node-id=0%3A1296&mode=design)
- [Figma mockups](https://www.figma.com/file/dBKlB23Hs2oa53euXcXmmK/526-toxic-exposure?type=design&node-id=0%3A411&mode=design&t=6jVvKzKPNp9o1VSn-1): `Refer to:` Figma page under "-----current workspace-----"
- [Epics/Issues](https://app.zenhub.com/workspaces/disability-experience-63dbdb0a401c4400119d3a44/issues/gh/department-of-veterans-affairs/va.gov-team/55513)

## Ticket Details

<details><summary> 🔎 Toggle to expand the intake ticket</summary>

### Which are you doing?
- [x] **Launching a new** page, tool, form, or app
- [ ] **Making changes** to an existing page, tool, form, or app

### Types of Support
What areas does your team need support in? Check all that apply.
#### Content
- [ ] Collaboration on content written by your team’s dedicated content writer 
- [x] Collaboration on content if your team does not have a dedicated content writer
#### Accessibility
- [x] Accessibility review of wireframes and/or prototypes
- [ ] Accessibility issue (general request)
- [ ] Accessibility PDF review request
- [ ] Accessibility user research support ([submit a separate, additional request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name))
- [ ] DST component or pattern 
- [ ] Something else 
#### IA
- [ ] User Flows
- [ ] Page placement
- [ ] URLs
- [ ] Breadcrumbs and Left Nav placement
- [ ] Redirects
#### Multilingual content
- [ ] Translating content
 
## Timeframe
_We guide and work alongside teams across nearly every OCTO digital product. We also work with partners across VA to lead content migration and manage all unauthenticated content on VA.gov. And we're always working to expand the VA.gov content style guide and our IA and accessibility documentation. We need to prioritize all intake requests based on our overall workload and VA and OCTO priorities._

_Tell us about your product's timeline. And we'll work with you to meet timeline needs as best we can._

### Have you worked with CAIA before? 
  - [x] Yes
  - [ ] No

### What's your team’s next step and its timing?

- **Our next step is** Updating design mocks with approved copy
- **Provide date if available**: xx/xx/xxxx

#### Timing for your next step: 
  - [x] Next week
  - [ ] This month
  - [ ] Next month
  - [ ] This quarter
  - [ ] Next quarter
  - [ ] 6 months
  - [ ] 1 year

### Is this timing related to a specific event or Congressionally mandated deadline?
  - [ ] Yes
  - [x] No

### If you're conducting research, when is that starting? 
**Provide date if available:** xx/xx/xxxx
  - [ ] Next week
  - [ ] This month
  - [ ] Next month
  - [ ] This quarter
  - [ ] Next quarter
  - [ ] 6 months
  - [ ] 1 year

### Will you release this new product incrementally (for example, release to 25% of users to start)? 
  - [ ] Yes
  - [x ] No
 
**Note**: _If you checked yes, we’ll reach out to discuss details about the content in the react widget (we use these widgets to display entry points for new products to a certain percentage of users who visit our static pages)._

  
### When do you expect to launch your product to 100% of users?

*Please provide an estimated date so our team can create other relevant tickets.*
- **We expect to launch our product to 100% of users on: Q1 of 2024
  
### Will this work be going through the Collaboration Cycle?
- [ x] Yes 
- [ ] No

### If no ...
- Skip to the _supporting artifacts_ section, below.
 
#### Please provide the link to your Collaboration Cycle ticket: 
- **Ticket (number)**: https://app.zenhub.com/workspaces/disability-experience-63dbdb0a401c4400119d3a44/issues/gh/department-of-veterans-affairs/va.gov-team/60140
- **Collab. Cycle Ticket Opened (date)**: 06/12/2023
 
#### Please check all of the phases in the Collaboration Cycle you have completed: 
- [x] Design Intent
- [x] Midpoint Review
- [ ] Staging
 
##### Design Intent
- [x] Accessibility
- [x] Content
- [ ] IA
- [ ] Translation
- [ ] Already completed, no assistance needed
- [ ] Unknown at this time
##### Midpoint
- [x] Accessibility
- [x] Content
- [ ] IA
- [ ] Translation
- [ ] Already completed, no assistance needed
- [ ] Unknown at this time
##### Staging
- [x] Accessibility
- [x] Content
- [ ] IA
- [ ] Translation
- [ ] Already completed, no assistance needed
- [ ] Unknown at this time
  
## Supporting artifacts
*Please provide links to supporting artifacts as available.*
- **Product outline**: https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/disability/526ez/README.md
- **User flows**: https://www.sketch.com/s/c353146f-eceb-4626-9918-7603c693417f
- **Prototype or mockups**: https://www.sketch.com/s/c353146f-eceb-4626-9918-7603c693417f/p/8E84DF49-0B6F-47DE-BF51-45BBBAD94FAB/canvas
- **If this is in Sketch, which version or specific page should we focus on?** WIP: Mock updates 12/2023
- **Content source of truth**: 
- **Research plan:**
- **Any epics/issues that may be helpful**: https://app.zenhub.com/workspaces/disability-experience-63dbdb0a401c4400119d3a44/issues/gh/department-of-veterans-affairs/va.gov-team/55513
- **Link to testing accounts** (please do not paste them in this ticket): 

### Have you already worked with someone from OCTO leadership?
- [ ] Danielle
- [ ] Beth
- [ ] Mikki
- [ ] Martha

### How do you plan to work with us on this request?
- [ ] I’ll take this to Beth or Danielle’s office hours.
- [x] I’d like to schedule time to talk about this.
- [ ] Let’s work async and meet as needed.
- [x] Schedule a kick-off call with your team and CAIA for your project **(Recommended)**

 
## Next steps

**Once you’ve submitted this ticket, please post a link to this issue in the [#sitewide-content-ia](https://dsva.slack.com/channels/sitewide-content-ia) Slack channel and tag `@Terry Nichols`.**

- If you also need engineering support from the Public Websites team, fill out their [intake request form](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E).
 
</details>

## Next Steps by Team 

```[tasklist]
### CAIA Ops
- [x] Initial Review - @coforma-terry and @strelichl
- [x] Setup kickoff meeting - scheduled for Wed Dec 13th at 12 PM ET  @coforma-terry and @strelichl
- [x] CAIA Kick Off meeting [internal CAIA notes](https://docs.google.com/document/d/1v9haSepdV3VTJeI4EHClVMW-IK3k_9X5xKhuJvk9Lyw/edit) and add recording to ticket -  @coforma-terry
- [x] [Link to kickoff recording 12/14](https://coformaco.zoom.us/rec/share/3yDh_zWWk-VRpOzzgXhSr_fEaddXVbF2d8WQgZmYYLEBsWgCouDnTFxzwmR9e1Km.UlrLSOUF5_7lTADJ) **Passcode:** uiZ9$hWd
- [x] Format Ticket: Adding Tasks, condensing view, tagging and updates @sara-amanda 12/14/2023
```
```[tasklist]
### CAIA a11y tasks
- [ ] **Sketch**: Review Files (All Pages)  [Sketch Prototype or mockups](https://www.sketch.com/s/c353146f-eceb-4626-9918-7603c693417f/p/8E84DF49-0B6F-47DE-BF51-45BBBAD94FAB/canvas): `Refer to:` WIP: Mock updates 12/2023
- [ ] **Review**: Connectivity between the `from` and `to` dates and the `not sure` button and how it impacts screen readers
- [ ] **Staging Artifact** A11ys available upon request to help once there is a coded prototype (VFS team starting post-KO
- [ ] **Future User Research**: `pending - none scheduled at this time` no ticket
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/72033
- [ ] CAIA Meeting: Content & A11y w/o 12/18/2023
```

```[tasklist]
### CAIA content tasks
- [x] **Sketch Files** - Initial Review - @katherine-fung reviewed purple text in Sketch prior to KO Meeting 12/14/2023
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/72012
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/78831
```

```[tasklist]
### CAIA IA tasks
- [ ] Possible URL change
```
```[tasklist]
### VFS Team Next Steps
- [x] **Attend** Kickoff Meeting with CAIA 12/13/2023
- [ ] Start the build
- [ ] Review Content Feedback
- [ ] Review IA Feedback
- [ ] Review A11y Feedback
- [ ] **Redirects and URLS**: If you need a page/URL redirected, a URL changed or a vanity URL set up, please submit a [Redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=jennymayoco%2C+kristinoletmuskat%2C+strelichl%2C+FranECross&labels=sitewide+CAIA%2C+Sitewide+IA%2C+Public+Websites%2C+VA.gov+frontend%2C+Redirect+request&projects=&template=redirect-request.md&title=Redirect+Request)
- [ ] **Experimental Design**: If you are creating an [experimental design,](https://design.va.gov/about/contributing-to-the-design-system/experimental-components-and-patterns#what-is-an-experimental-component-or-pattern?) please [also contact the DST. ](https://design.va.gov/about/contributing-to-the-design-system/suggest-an-addition-or-update)
- [ ] **User Research**: [Submit Research Ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name)
- [ ] The [#accessibility-help channel](https://dsva.slack.com/archives/C8E985R32) in the DSVA Slack, is also available to you as a resource. Everyone is welcome to ask questions or get help from our a11y (accessibility) specialists. Tag `@Terry Nichols` to get a11y help asap.
```

**Comment by pacerwow** [comment]
Closing as we went back to Midpoint review for Toxic Exposure in February. When we go to Staging Review in July, CAIA will have an opportunity to review content again and give feedback.
